src/libsync/configfile.cpp

```diff
@@ -93,7 +93,7 @@ const QString moveToTrashC() { return QStringLiteral("moveToTrash"); }
 }
 
 QString ConfigFile::_confDir = QString();
-const std::chrono::seconds DefaultRemotePollInterval { 30 }; // default remote poll time in milliseconds
+const std::chrono::seconds DefaultRemotePollInterval { 30 }; // default remote poll time in seconds
```

Comment:
I think you can just get rid of "in seconds", since that's clear from the type. Should one ever change the type to, e.g., `std::chrono::milliseconds` or `std::chrono::minutes`, the comment would have to be touched again.

Reply:
After removing "in seconds" from the comment, it ended up pretty much repeating the words that are already in the const name, which is more redundant comment crud that can get out-of-sync with the real code. So I removed the whole comment.